packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -553,6 +553,7 @@ type MessagesType = {
   'copy.year': '1Y'
   'copy.or': 'or'
   'copy.over_your_limit': 'Over Your Limit'
+  'copy.processing': 'Processing...'
```

Comment:
i think it might be better to have just the copy in here. `'copy.processing': 'Processing'`. You can then add the periods in the component itself